### PR TITLE
Merging Dev Branch Changes to Master

### PR DESCRIPTION
**Pull request by riacheruvu**

### CODE REVIEW

**Comment by adrianboguszewski** [comment]
Thanks for this PR @riacheruvu 

Let's do the following:
1. Replace the PDF document with the Wiki page and provide the link to that in the main README file
2. Split this PR into 2, each containing just one demo and the related files

**Review by adrianboguszewski — COMMENTED** [review]
Thanks for this PR @riacheruvu

Let's do the following:

Replace the PDF document with the Wiki page and provide the link to that in the main README file
Split this PR into 2, each containing just one demo and the related files

**Comment by riacheruvu** [comment]
Thank you, @adrianboguszewski! I've done so, and will create the second demo PR once this one is approved. I will be creating a separate PR for the README as it involves more restructuring.

File: demos/onnxruntime_yolov8_demo/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ultralytics==8.2.81
```
Comment (adrianboguszewski) [inline demos/onnxruntime_yolov8_demo/requirements.txt:1]: jupyterlab is missing here

File: demos/onnxruntime_yolov8_demo/YOLOv8 Object Detection with ONNX and OpenVINO Execution Provider.ipynb [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "[notice] A new release of pip is available: 24.0 -> 24.2\n",
+      "[notice] To update, run: python.exe -m pip install --upgrade pip\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip install ultralytics"
```
Comment (adrianboguszewski) [inline demos/onnxruntime_yolov8_demo/YOLOv8 Object Detection with ONNX and OpenVINO Execution Provider.ipynb:78]: It's already installed in requirements

File: demos/onnxruntime_yolov8_demo/YOLOv8 Object Detection with ONNX and OpenVINO Execution Provider.ipynb [hunk 3]
```diff
@@ -0,0 +1,448 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "id": "WrPPbh0Qgnt-"
+   },
+   "outputs": [],
+   "source": [
+    "original_model_path = \"/content/model/yolov8m.onnx\"\n",
```
Comment (adrianboguszewski) [inline demos/onnxruntime_yolov8_demo/YOLOv8 Object Detection with ONNX and OpenVINO Execution Provider.ipynb:268]: the model doesn't exists under this path

**Review by adrianboguszewski — CHANGES_REQUESTED** [review]

**Review by adrianboguszewski — CHANGES_REQUESTED** [review]
Ok. Everything works. So my only comment is to improve the README file. Please add there instructions on how to install/run it. A screenshot presenting demo would be also nice. We need something similar to: https://github.com/openvinotoolkit/openvino_build_deploy/blob/master/demos/onnxruntime_lcm_demo/README.md

**Comment by riacheruvu** [comment]
> Ok. Everything works. So my only comment is to improve the README file. Please add there instructions on how to install/run it. A screenshot presenting demo would be also nice. We need something similar to: https://github.com/openvinotoolkit/openvino_build_deploy/blob/master/demos/onnxruntime_lcm_demo/README.md

Completed! @adrianboguszewski

File: demos/onnxruntime_yolov8_demo/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+- To activate the virtual environment use
+  ```sh
+  \<venv-name>\Scripts\activate
+  ```
+- Install the required dependencies via pip
+  ```sh
+  pip install -r requirements.txt
+  ```
+- Now you only need a Jupyter server to start.
+- For details, please refer to [Installation Guide](../../README.md#-installation-guide).
```
Comment (adrianboguszewski) [inline demos/onnxruntime_yolov8_demo/README.md:25]:
```suggestion
```

**Review by adrianboguszewski — APPROVED** [review]

**Pull request merged**